src/data/pokemon/evolution.h

```diff
@@ -36,6 +36,9 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_MEOWTH]                = {{EVO_LEVEL, 28, SPECIES_PERSIAN}},
     [SPECIES_PSYDUCK]               = {{EVO_LEVEL, 33, SPECIES_GOLDUCK}},
     [SPECIES_MANKEY]                = {{EVO_LEVEL, 28, SPECIES_PRIMEAPE}},
+#if P_GEN_9_POKEMON == TRUE
+    [SPECIES_PRIMEAPE]              = {{EVO_NONE, 0, SPECIES_ANNIHILAPE}},
```

Comment:
I think it would be fine to evolve with Rage Fist.

Reply:
I would also add Psyshield Bash as evo move to Stantler sind it's an egg move.

Reply:
> I think it would be fine to evolve with Rage Fist.
I would rather we add the actual evo methods instead of temporary ones like this

Reply:
> > I think it would be fine to evolve with Rage Fist.
> 
> I would rather we add the actual evo methods instead of temporary ones like this

Me too but I don't think it's currently possible

Reply:
> I would also add Psyshield Bash as evo move to Stantler sind it's an egg move.

Already added in https://github.com/rh-hideout/pokeemerald-expansion/pull/3547/commits/d43eaa0df77e561bcc60485b53ef8a19ea052fa0!